pkg/apis/deployment/v1/member_status.go

```diff
@@ -76,6 +76,8 @@ type MemberStatus struct {
 	Image *ImageInfo `json:"image,omitempty"`
 	// OldImage holds old image defails
 	OldImage *ImageInfo `json:"old-image,omitempty"`
+	// Architecture defines Image architecture type
+	Architecture ArangoDeploymentArchitectureType `json:"architecture,omitempty"`
```

Comment:
Why ArangoDeploymentArchitectureType instead of *ArangoDeploymentArchitectureType?

It wil lbe easier to determine if arch is set.

Reply:
done